elixir/apps/domain/lib/domain/types/ip_port.ex

```diff
@@ -88,4 +88,11 @@ defmodule Domain.Types.IPPort do
     ip = ip |> :inet.ntoa() |> List.to_string()
     "[#{ip}]:#{port}"
   end
+
+  def put_default_port(ip_port, default_port) do
+    port = ip_port.port || default_port
+    %{ip_port | port: port}
+  end
+
+  def default_dns_port, do: 53
```

Comment:
I think this one should be in [clients_upstream_dns.ex](https://github.com/firezone/firezone/pull/2432/files#diff-26148f72d6ac674d1838d9ccf548861b309559f6ba53fadeaf7f4b24bfe609f5)

Reply:
I was torn on this one since it is the true default port for DNS it could relate to more things than just the `ClientsUpstreamDNS`, but I'm also fine to move it.

Reply:
This has been moved.